projects/ltc7871/README.rst

```diff
@@ -0,0 +1,185 @@
+
+**Connections**:
+
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|  Function                       | MAX32690EVKIT Pin            |        Voltage shifter       |     DC2886A Pin           |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|SPI Clock (SCK)                  | P2.29 (SPI0B_SCK)            |                              |           SCL             |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|SPI Master In Slave Out (MISO)	  | P2.27 (SPI0B_MISO)	         |             A1               |                           |
++                                 +------------------------------+------------------------------+---------------------------+
+|                                 |                              |             B1               |           SDO*            |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|SPI Chip Select (CS)             | P2.26 (SPI0B_SS0) 	         |                              |           CSB             |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|SPI Master Out Slave In (MOSI)	  | P2.28 (SPI0B_MOSI)           |                              |           SDA             |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|GPIO (PWMEN Pin)                 | P4.0 (GPIO)       	         |                              |           PWMEN           |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|Ground (GND)                     | GND               	         |             GND              |           GND             |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|3.3V                             | Vaux                         |             LV               |                           |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+|5V                               |                              |             HV               |            V5             |
++---------------------------------+------------------------------+------------------------------+---------------------------+
+
+'* - DC2886A SDO's external pull-up resistor should be replaced with 1kohm resistor
+
+**Build Command**
+
+.. code-block:: bash
+
+	# to delete current build
+	make PLATFORM=maxim TARGET=max32690 reset
+	# to build the project and flash the code
+	make PLATFORM=maxim TARGET=max32690 run
```

Comment:
maybe split this in two, dont build and run on the same line, it works but may confuse readers